plugins/CoreHome/angularjs/reporting-menu/reportingmenu.controller.js

```diff
@@ -235,6 +262,12 @@
 
             $('#loadingError').hide();
 
+            if ($scope.currentSubcategory != 'Live_VisitorLog') {
```

Comment:
Might make sense to also _not_ show it on the live map

Reply:
👍

Reply:
also the real time widget page. eventually would need to maybe let plugins hook into this somehow